cli/tests/echo_server.ts

```diff
@@ -1,4 +1,4 @@
-const addr = Deno.args[0] || "0.0.0.0:4544";
+const addr = Deno.args[1] || "0.0.0.0:4544";
```

Comment:
Of course you don't have to include this but I also don't understand why not. 🤷‍♀️ 
I mean this is just a clear typo...